samples/test/downscoping-with-cab.test.js

```diff
@@ -30,9 +30,9 @@ const {promisify} = require('util');
 const exec = promisify(cp.exec);
 // Copy values from the output of samples/scripts/downscoping-with-cab-setup.js.
 // GCS bucket name.
-const bucketName = 'cab-int-bucket-z2zsauf4sj';
+const bucketName = 'cab-int-bucket-brd3qlsuok';
 // GCS object name.
-const objectName = 'cab-first-z2zsauf4sj.txt';
+const objectName = 'cab-first-"brd3qlsuok.txt';
```

Comment:
curious why we changed this object name.

Reply:
Good question: The bucket and object previously didn’t exist - and the test has silently failed until this PR as we were asserting a string that existed in the error message (‘first’).